e2e/drone_setup.sh

```diff
@@ -10,15 +10,22 @@ fi
 cd e2e/digitalocean
 terraform init
 
-until terraform apply -auto-approve -var "cluster_name=${DRONE_BUILD_NUMBER}-do" -var "image=$1"
+WORKER_UP=${WORKER_UP:-false}
+if [ "${WORKER_UP}" = "true" ]; then
+  WORKER_UP_COUNT="1"
```

Comment:
Maybe it could just have this variable in the ENV config.

Reply:
Done.